application/controllers/Qslprint.php

```diff
@@ -92,42 +92,54 @@ public function exportcsv()
 		$file = fopen('php://output', 'w');
 
 		$header = array("STATION_CALLSIGN",
-						"COL_CALL",
-						"COL_QSL_VIA",
-						"COL_TIME_ON",
-						"COL_MODE",
-						"COL_SUBMODE",
-						"COL_FREQ",
-						"COL_BAND",
-						"COL_RST_SENT",
-						"COL_SAT_NAME",
-						"COL_SAT_MODE",
-						"COL_QSL_RCVD",
-						"COL_COMMENT",
-						"COL_ROUTING",
+						"CALL",
+						"QSL_VIA",
+						"DATE_ON",
+						"TIME_ON",
+						"MODE",
+						"SUBMODE",
+						"FREQ",
+						"BAND",
+						"RST_SENT",
+						"SAT_NAME",
+						"SAT_MODE",
+						"PROPAGATION",
```

Comment:
```suggestion
						"PROP_MODE",
```

Better use the ADIF name here? (though ROUTING is not either)